packages/blocks/cloudhealth/src/lib/common/call-rest-api.ts

```diff
@@ -15,7 +15,7 @@ export async function makeGetRequest<T = unknown>(
     },
     queryParams,
   });
-  return response.body;
+  return response;
```

Comment:
returning the full response, since the headers contain the pagination data

<img width="278" height="58" alt="Image" src="https://github.com/user-attachments/assets/dd44f94b-6771-4fda-ae53-47791086093b" />